pom.xml

```diff
@@ -61,17 +57,26 @@
         <dependency>
             <groupId>com.h2database</groupId>
             <artifactId>h2</artifactId>
-            <scope>test</scope>
+             <scope>test</scope>
+         </dependency>
+         <dependency>
+             <groupId>org.postgresql</groupId>
+             <artifactId>postgresql</artifactId>
+             <version>9.4-1206-jdbc42</version>
```

Comment:
This version of the driver won't work with PostgreSQL 10 (which is being adopted now). Could it be upgraded to 42.2.5?

Reply:
fixed in c973edc